.github/workflows/ci-cd.yaml

```diff
@@ -375,6 +380,7 @@ jobs:
           set -ex
           for target_host in ${TRENTO_AGENT_HOSTS//,/ }
           do
+            ssh "$TRENTO_USER@$target_host" "zypper in -y golang-github-prometheus-node_exporter" 
```

Comment:
By the way @rtorrero ,
Here we would need to start the prometheus systemd daemon, right?
`systemctl start prometheus-node_exporter`

Reply:
Not really, it will be automatically started with the `sudo systemctl enable --now trento-agent.service` that we have on line :385.

Notice that as you said; this will make node_exporter a hard dependency for trento, causing trento-agent.service to fail when node_exporter doesn't get started.

Reply:
Ok, thanks @rtorrero ,
I thought that the `tgz` installation wasn't installing the service file. But yes, it does hehe
Perfect them!